.travis.yml

```diff
@@ -36,7 +36,7 @@ script:
   - flake8 --version
   - flake8 src
   - flake8 --exclude=src,docs --extend-ignore=D
-  - pytest --cov=dowel
+  - pytest -n $(nproc) --cov=dowel
```

Comment:
I believe this can also just be `-n auto`, according to the pytest-xdist documentation.